### PR TITLE
add test cases

### PR DESCRIPTION
**Pull request by bprize15**
resolves https://github.com/oncokb/oncokb-pipeline/issues/671

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -13,12 +13,16 @@ jobs:
     # A list of available CircleCI Docker Convenience Images are available here: https://circleci.com/developer/images/image/cimg/openjdk
     docker:
       - image: cimg/openjdk:11.0
+      - image: genomenexus/gn-mongo:0.30
```
Comment (leexgh) [inline .circleci/config.yml:16]: Do we want to stick to this version?

**Review by leexgh — COMMENTED** [review]
Do we need 1000+ test cases? Can we only include some general cases and corner cases? My browser is freezing on viewing this so can't load all tests lol, not sure how you picked these cases

File: .circleci/config.yml (hunk 1 above)
Comment (bprize15) [reply .circleci/config.yml:16]: Yeah that's my plan since we need to pick a fixed version

**Review by leexgh — APPROVED** [review]
Thank you!
